awx/main/models/events.py

```diff
@@ -731,6 +734,7 @@ class Meta:
             ('inventory_update', 'end_line'),
         ]
 
+    id = models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')
```

Comment:
I didn't bother doing this migration for system job events, because I can't imagine a universe in which you'd ever spawn 2B+ of these.

Reply:
i guess that depends on whether or not cleanup_jobs has every delete as its own event? If it does, you're effectively doubling the event every cleanup

Reply:
Yea, you're right.  I've added `systemjobevent` to this list; it wouldn't hurt to go ahead and migrate it as well.